demo/box.py

```diff
@@ -22,3 +22,42 @@ def app():
     ):
       me.text(text="hi1")
       me.text(text="hi2")
+
+    with me.box(
+      style=me.Style(
+        background="blue",
+        height=50,
+        margin=me.Margin.all(16),
+        border=me.Border.all(
+          me.BorderSide(width=2, color="yellow", style="dotted")
+        ),
+        border_radius=10,
+      )
+    ):
+      me.text(text="Example with all sides bordered")
+
+    with me.box(
+      style=me.Style(
+        background="purple",
+        height=50,
+        margin=me.Margin.symmetric(vertical=24, horizontal=12),
+        border=me.Border(
+          top=me.BorderSide(width=4, color="white", style="double"),
+          bottom=me.BorderSide(width=4, color="white", style="double"),
+        ),
+      )
+    ):
+      me.text(text="Example with top and bottom borders")
+
+    with me.box(
+      style=me.Style(
+        background="cyan",
+        height=50,
+        margin=me.Margin.symmetric(vertical=24, horizontal=12),
+        border=me.Border(
```

Comment:
[nit] Border has a static method called symmetric. So you can do a shortcut:

```
me.Border.symmetric(horizontal=me.BorderSide(width=2, color="black", style="groove"))
```

Reply:
Basicalyl did this
`
  with me.box(
      style=me.Style(
        background="cyan",
        height=50,
        margin=me.Margin.symmetric(vertical=24, horizontal=12),
        border=me.Border.symmetric(horizontal=me.BorderSide(width=2, color="black", style="groove"))
      )
    ):
      me.text(text="Example with left and right borders")
`